src/command-line/install.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const colors = require("colors/safe");
+const program = require("commander");
+const Helper = require("../helper");
+const Utils = require("./utils");
+
+program
+	.command("install <package>")
+	.description("Install a package")
+	.on("--help", Utils.extraHelp)
+	.action(function(packageName) {
+		const fs = require("fs");
+		const fsextra = require("fs-extra");
+		const path = require("path");
+		const child = require("child_process");
+		const packageJson = require("package-json");
+
+		if (!fs.existsSync(Helper.CONFIG_PATH)) {
+			log.error(`${Helper.CONFIG_PATH} does not exist.`);
+			return;
+		}
+
+		packageJson(packageName, {
+			fullMetadata: true
+		}).then((json) => {
+			if (!("lounge" in json)) {
+				log.error(`${colors.red(packageName)} does not have The Lounge metadata.`);
+
+				process.exit(1);
+			}
+
+			log.info(`Installing ${colors.green(packageName)}...`);
+
+			const packagesPath = Helper.getPackagesPath();
+			const packagesParent = path.dirname(packagesPath);
+			const packagesConfig = path.join(packagesParent, "package.json");
+
+			// Create node_modules folder, otherwise npm will start walking upwards to find one
+			fsextra.ensureDirSync(packagesPath);
+
+			// Create package.json with private set to true to avoid npm warnings
+			fs.writeFileSync(packagesConfig, JSON.stringify({
+				private: true,
+				description: "Packages for The Lounge. All packages in node_modules directory will be automatically loaded.",
```

Comment:
Just for the record, `description` is not needed, but I get why you add it. I'm fine with it.